docs/kratos/organizations/organizations.mdx

```diff
@@ -169,9 +169,15 @@ Some notes on the fields of the JSON payload:
 - `mapper_url` is the URL to a JSONnet file that maps the OIDC provider's claims to Ory's identity schema. You can use the
   `base64` scheme to embed the JSONnet file directly in the JSON payload.
 
+:::tip
```

Comment:
Could you instead explain PKCE briefly here in the JSON patch above? So add "pkce" parameter and explain it in the list below. You can link to the new doc for more info

Reply:
done